tests/kafkatest/services/console_consumer.py

```diff
@@ -184,9 +183,12 @@ def start_cmd(self, node):
 
         # LoggingMessageFormatter was introduced after 0.9
         if node.version > LATEST_0_9:
-            cmd+=" --formatter kafka.tools.LoggingMessageFormatter"
+            cmd += " --formatter kafka.tools.LoggingMessageFormatter"
+
+        # enable systest events is only available in 0.10.0 and later
+        if node.version >= V_0_10_0_0:
```

Comment:
`V_0_10_0_0` is defined independently of any kafkatest version bump. kafkatest version is the version of the kafkatest module, whereas an instance of `KafkaVersion` refers to a version of Kafka.

It's true that `KafkaVersion` inspects the __init__ file to figure out what `TRUNK` should mean, but it strips any `dev*` suffix, so if we're in 0.10.0 branch, `TRUNK` actually equals `V_0_10_0_0`